{{ cookiecutter.project_name }}/README.md

```diff
@@ -0,0 +1,141 @@
+\[!\[Actions status\](https://github.com/elixir-cloud-aai/Python
+Cookiecutter/workflows/CI/badge.svg)\](https://github.com/elixir-cloud-aai/Python
+Cookiecutter/actions)
+[![Bandit](https://img.shields.io/badge/security-bandit-yellow.svg)](https://bandit.readthedocs.io/en/latest/)
+\[!\[codecov\](https://codecov.io/gh/elixir-cloud-aai/Python
+Cookiecutter/branch/main/graph/badge.svg)\](https://codecov.io/gh/elixir-cloud-aai/Python
+Cookiecutter)
+[![Documentation Status](https://readthedocs.org/projects/python_cookiecutter/badge/?version=latest)](https://python_cookiecutter.readthedocs.io/en/latest/?badge=latest)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](./LICENSE)
+[![Python 3.11](https://img.shields.io/badge/python-3.11-blue.svg)](https://www.python.org/downloads/release/python-311/)
+\[!\[GitHub
+contributors\](https://img.shields.io/github/contributors/elixir-cloud-aai/Python
+Cookiecutter)\](https://github.com/elixir-cloud-aai/Python
+Cookiecutter/graphs/contributors)
+[![Ruff](https://img.shields.io/badge/linter%20&%20formatter-ruff-000000.svg)](https://docs.astral.sh/ruff/)
+[![Safety](https://img.shields.io/badge/security-safety-orange.svg)](https://safetycli.com/product/safety-cli)
+
+# python_cookiecutter
+
+A short description of the project.
```

Comment:
Shouldn't we use template vars to fill this in with the project title and description?

Reply:
oh crap!! Done :+1: